package.json

```diff
@@ -14,14 +14,29 @@
     "url": "https://github.com/focuslocus/purpose/issues"
   },
   "homepage": "https://github.com/focuslocus/purpose#readme",
+  "scripts": {
+    "start": "node ./bin/www",
```

Comment:
We currently use this for the dev server. Any chance we can incorporate webpack dev server into our server file?